code/__DEFINES/DNA.dm

```diff
@@ -65,30 +65,23 @@
 #define NOBLOOD 6
 #define NOTRANSSTING 7
 #define NOZOMBIE 8
-/// Uses weird leg sprites. Optional for Lizards, required for ashwalkers. Don't give it to other races unless you make sprites for this (see human_parts_greyscale.dmi)
-#define DIGITIGRADE 9
-#define NO_UNDERWEAR 10
-#define NOSTOMACH 11
-#define NO_DNA_COPY 12
-#define DRINKSBLOOD 13
-/// Use this if you want to change the race's color without the player being able to pick their own color. AKA special color shifting
-#define DYNCOLORS 14
-#define AGENDER 15
-/// Do not draw eyes or eyeless overlay
-#define NOEYESPRITES 16
-/// Used for determining which wounds are applicable to this species.
-/// if we have flesh (can suffer slash/piercing/burn wounds, requires they don't have NOBLOOD)
-#define HAS_FLESH 17
-/// if we have bones (can suffer bone wounds)
-#define HAS_BONE 18
-///If we have a limb-specific overlay sprite
-#define HAS_MARKINGS 19
-/// Do not draw blood overlay
-#define NOBLOODOVERLAY 20
-///No augments, for monkeys in specific because they will turn into fucking freakazoids https://cdn.discordapp.com/attachments/326831214667235328/791313258912153640/102707682-fa7cad80-4294-11eb-8f13-8c689468aeb0.png
-#define NOAUGMENTS 21
-///will be assigned a universal vampire themed last name shared by their department. this is preferenced!
-#define BLOOD_CLANS 22
+//PARIAH MODULAR EDIT START
```

Comment:
PSA

This isn't modular. Modular would be putting it in its own folder. `// PARIAH EDIT START` would make much more sense here.

Reply:
Ah, I see I see. Thanks for the tip, ~~I'm not that good at this stuff as you can tell lmao~~